analyze-network-with-subnetwork-trace/src/main/res/layout/activity_main.xml

```diff
@@ -277,7 +277,7 @@
                     android:layout_marginTop="8dp"
                     android:layout_marginEnd="16dp"
                     android:layout_marginBottom="8dp"
-                    android:background="@drawable/border"
```

Comment:
This line could be added back but will need to remove the solid background from the drawable. 
```xml
<solid android:color="#A9E8E8E8" />
```